FrontEnd/src/components/ProfileDetail/MainInfo/ProfileDetailNavBar.jsx

```diff
@@ -10,38 +14,66 @@ const MENU_LINKS = {
     'cooperation': 'Формат співпраці',
 };
 
-function ProfileDetailNavBar() {
-
+function ProfileDetailNavBar({ data }) {
   const { hash } = useLocation ();
+  const navigate = useNavigate ();
+  const { dataInComponents } = useContext(DataContext);
+  const [activeLink, setActiveLink] = useState('');
+
+  useEffect(() => {
+    if (hash) {
+      setActiveLink(hash.substring(1));
+    } else {
+      setActiveLink(
+        data.is_registered && dataInComponents.includes('about-company') ? 'about-company' :
```

Comment:
I would recommend using an object with keys instead of the array of strings. 
`
{
  'about-company': true
  ...
}
`